builder/kubernetes/build.go

```diff
@@ -383,13 +383,13 @@ func (b *kubernetesBuilder) buildContainerImage(ctx context.Context, app *apptyp
 		}
 
 		// Default to web process name and entrypoint and cmd from container
-		if len(processes) == 0 {
+		if shouldDefaultToImageCommands(processes) {
```

Comment:
@ravilock we may need a else statement to cover some cases

Reply:
Tell me more

Reply:
@ravilock i think that you could simplify to if len(finalProcesses) == 0, that means that Procfile and tsuru.yaml does not have any command